### PR TITLE
WIP handle compat with ember-cli 0.2+

### PR DESCRIPTION
**Pull request by hashrocketeer**

### CODE REVIEW

**Comment by hashrocketeer** [comment]
We found a possible issue with the registry in EmberCLI not respecting the order defined in the package.json correctly.  We are working on a test case for it now.  Once the fix has been made there this will likely work "out of the box"™  We'll post back as soon as we have more information. 

/cc @bantic @mixonic

**Comment by mixonic** [comment]
@hashrocketeer thanks for the update! :+1:

**Comment by mphasize** [comment]
The registry issue in ember-cli 2.0 just got fixed. :-)
https://github.com/ember-cli/ember-cli/issues/3396

File: package.json [hunk 1]
```diff
@@ -39,9 +39,12 @@
...
     "configPath": "tests/dummy/config",
-    "before": [ "ember-cli-htmlbars"]
+    "before": [
+      "ember-cli-htmlbars"
+    ]
   },
   "dependencies": {
-    "broccoli-filter": "^0.1.11"
+    "broccoli-filter": "^0.1.11",
+    "ember-cli-version-checker": "^1.0.2"
```
Comment (atsjj) [inline package.json]: Also needs this: `"emblem": "0.5.0"`

Didn't work for me when I tried this PR without this dep.

Comment (mixonic) [inline package.json]: Ah, that is probably the rebase conflict. It is correct in master: https://github.com/201-created/ember-cli-emblem-hbs-printer/blob/master/package.json#L48 Would definitely take another PR with it rebased and tested in a real app ;-)

**Comment by mixonic** [comment]
closed in favor of https://github.com/201-created/ember-cli-emblem-hbs-printer/pull/3

**Pull request closed without merging**